src/screens/Feed.js

```diff
@@ -45,87 +67,29 @@ export default class FeedScreen extends React.Component {
             username={username}
             commentsLength={comments.length}
           />
+          <Divider />
         </View>
       </TouchableWithoutFeedback>
     );
   };
 
   //sample post data for ui testing purposes. We also have use react-native carousel for swiping posts.
   render() {
-    const posts = [
-      {
-        id: 0,
-        img:
-          'https://www.familyandmedia.eu/wp-content/uploads/2018/10/529382-4386816-selfie-psicologia-725x545.jpg',
-        caption:
-          'Quarantine selfie #coronasa dsajl as lnsaln aln flsan lsan lfaslf fnasl nfals nlsan lsan lasn lfasn lfasnf lan',
-        likes: 2,
-        saves: 0,
-        comments: [
-          {
-            userName: 'Model1234',
-            userImage:
-              'https://images.unsplash.com/photo-1534528741775-53994a69daeb?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80',
-            comment: 'Slay girllll!!!',
-          },
-        ],
-        username: 'gabriela',
-        likers: {
-          dennyliang: true,
-        },
-        bookmarkers: {
-          dennyliang: false,
-        },
-      },
-      {
-        id: 1,
-        img:
-          'https://www.familyandmedia.eu/wp-content/uploads/2018/10/529382-4386816-selfie-psicologia-725x545.jpg',
-        caption: 'Quarantine selfie #corona',
-        likes: 2,
-        saves: 0,
-        comments: [
-          {
-            userName: 'Model1234',
-            userImage:
-              'https://images.unsplash.com/photo-1534528741775-53994a69daeb?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80',
-            comment: 'Slay girllll!!!',
-          },
-        ],
-        username: 'gabriela',
-        likers: {
-          dennyliang: false,
-        },
-        bookmarkers: {},
-      },
-      {
-        id: 2,
-        img:
-          'https://www.familyandmedia.eu/wp-content/uploads/2018/10/529382-4386816-selfie-psicologia-725x545.jpg',
-        caption: 'Quarantine selfie #corona',
-        likes: 2,
-        saves: 0,
-        comments: [
-          {
-            userName: 'Model1234',
-            userImage:
-              'https://images.unsplash.com/photo-1534528741775-53994a69daeb?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80',
-            comment: 'Slay girllll!!!',
-          },
-        ],
-        username: 'gabriela',
-        likers: {},
-        bookmarkers: {
-          dennyliang: true,
-        },
-      },
-    ];
     return (
-      <FlatList
-        data={posts}
-        renderItem={this.renderPosts}
-        keyExtractor={item => item.id}
-      />
+      <SafeAreaView style={{ flex: 1 }} >
+        <Swiper
```

Comment:
How will swiper work with worth rendering posts? Ie is this a component we are using now to get Screenshot detecter to work or will we have to switch over back to FlatList so that when we are making post rendering calls, we don't pre-load 100 images at once?